portfolio/src/main/webapp/script.js

```diff
@@ -1,3 +1,20 @@
 function changeImage() {
   document.getElementById("switch").className = "index-image-2"; 
-}
\ No newline at end of file
+}
+
+function createListElement(text) {
+  const liElement = document.createElement('li');
```

Comment:
Say listItemElement or listElement. Try to avoid abbreviating words if you can. (Even though the tag is actually ```<li>```).